### DISCUSSION TITLE
Advance setting Watt offset

### DISCUSSION THREAD

**Discussion by Barnello83** [cagnulein/qdomyos-zwift#1714] [Ideas]
Hi,

it would be great if the app permits to put positive offset in Advance setting “watt offset”.

This in order to compare stryd wattage indoor whit outdoor.

Thank you in advice

Federico

**Comment by cagnulein** [comment]
sure i will do tomorrow;)

Il giorno mer 18 ott 2023 alle 20:53 Barnello83 ***@***.***>
ha scritto:

**Comment by cagnulein** [comment]
done @Barnello83 I will send you a new version later today!

**Comment by Barnello83** [comment]
Great @cagnulein!!

Thank you!!

**Comment by cagnulein** [comment]
version sent! let me know!

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno gio 19 ott 2023 alle ore 08:47 Barnello83 <
***@***.***> ha scritto:

**Comment by Barnello83** [comment]
Hi,

debug log of yesterday activity and this morning activity.


[debug-Tue_Oct_24_22_11_10_2023.log](https://github.com/cagnulein/qdomyos-zwift/files/13163413/debug-Tue_Oct_24_22_11_10_2023.log)

[debug-Wed_Oct_25_06_49_47_2023.zip](https://github.com/cagnulein/qdomyos-zwift/files/13163454/debug-Wed_Oct_25_06_49_47_2023.zip)

**Reply by cagnulein** [reply]
hi @Barnello83 the problem here is that you set the watt gain to 0. It must always be a value different than 0 otherwise the resulted watt will be always 0 (watt = raw_wattage * gain). So please set it to 1 and se the watt offset to 40
Let me know

**Reply by Barnello83** [reply]
Hi @cagnulein thank you very much!!

I have done a mess🤣!!!

I’ll update you as soon as possible!!

**Reply by Barnello83** [reply]
Hi @cagnulein perfect!!

It’s my fault, everything works fine!!

Thank you

**Comment by Barnello83** [comment]
Hi @cagnulein 
I would like to ask you if it’s possible to use floating number to speed offset.

I have tried, but I can use only integers.

Thank you

Federico

**Reply by cagnulein** [reply]
i checked and it's already a float. You just need to copy and paste the float value from another app.

for example write 10.7 in an app, copy the value and paste it in the speed offset setting in qz and press ok.
I also already fixed the fact that you can't input it directly.
let me know

**Reply by Barnello83** [reply]
With copy and paste it works.
Directly from the app I continue to not have the possibility to insert. I can see only the pad without the point.
Have I to re-download the app?In TestFlight I see the same version that I have already installed

**Reply by cagnulein** [reply]
Great!
No I just did the modification in the code, I will upload it later :)

**Comment by cagnulein** [comment]
sure i will do this

Il giorno sab 9 dic 2023 alle 16:39 Barnello83 ***@***.***>
ha scritto:

**Reply by Barnello83** [reply]
@cagnulein 
I have another question:
is possible to set the power zone using 5 zones (like stryd) instead of 7?
Is also possible, like the hearth rate, to set the zone using custom percentage?

thank’s in advice 

Federico

**Comment by cagnulein** [comment]
heart zone are customizable in the settings
power zones are doing in this way to stay aligned to peloton. i would like
to keep as they are for now

Il giorno dom 10 dic 2023 alle 16:14 Barnello83 ***@***.***>
ha scritto:
